doc/Data Plane Code Generation/Sample.md

```diff
@@ -0,0 +1,55 @@
+Samples are where developers could start to learn a library. You should generate samples scaffold by running the quick start [tool](AzureSDKCodeGeneration_DataPlane_Quickstart.md#create-starter-package). Different scenarios should be separated by different files. The samples should match the structure containing the following key features:
+```
+├── ...
+├── samples
+│   ├── README.md
+│   ├── Sample1_{Scenario1}.md
```

Comment:
what shall we include in the md file?

Reply:
I explained in the blow section `Folder samples`.